src/main/scala/io/iohk/ethereum/db/dataSource/RocksDbDataSource.scala

```diff
@@ -0,0 +1,175 @@
+package io.iohk.ethereum.db.dataSource
+
+import java.util.concurrent.locks.ReentrantReadWriteLock
+
+import org.rocksdb._
+
+class RocksDbDataSource(private var db: RocksDB, private val rocksDbConfig: RocksDbConfig) extends DataSource {
```

Comment:
ethereumj and rocks docs mentions need to load  rocks db library with `RocksDB.loadLibrary();`  - https://github.com/ethereum/ethereumj/blob/develop/ethereumj-core/src/main/java/org/ethereum/datasource/rocksdb/RocksDbDataSource.java#L69 
Dont we need something like this ?